libbeat/processors/timestamp/timestamp.go

```diff
@@ -84,6 +84,20 @@ func newFromConfig(c config) (*processor, error) {
 	return p, nil
 }
 
+func loadLocation(timezone string) (*time.Location, error) {
+	timezoneFormats := []string{"-07", "-0700", "-07:00"}
```

Comment:
Should timezoneFormats here be a global variable? (No need to change it if you prefer this.)

Reply:
Will we support `Z0700   Z07:00` as well? I was reading https://yourbasic.org/golang/format-parse-string-time-date-example/ to see what are the offset/timezone format there are.

Reply:
I saw this format in the Go documentation, I think this is only useful in complete timestamps to format times with numeric zero timezones (UTC) as `Z`, or to parse `Z` as zero timestamps.

In our case here we are not parsing complete timestamps, but only timezones, that come from configuration. We could use these formats to support `Z` as a valid value for the timezone setting (as `timezone: 'Z'`), but I am not sure if we want to do that, specially when there are more usual alternatives that will work, as `timezone: 'Etc/UTC'`, or `timezone: '+0000'`.